pages/_app.jsx

```diff
@@ -0,0 +1,29 @@
+import '../styles/globals.css';
+import React, { useEffect } from 'react';
+import * as Sentry from '@sentry/react';
+import i18n from '../i18n';
+
+Sentry.init({
+  dsn: process.env.SENTRY_DSN_KEY,
+  integrations: [
+    new Sentry.BrowserTracing({
+      tracePropagationTargets: ['localhost', /^https:\/\/yourserver\.io\/api/],
```

Comment:
It will be better to use the environment variables for this links from the .env file. You can check [this library](https://www.npmjs.com/package/dotenv)